dspace-api/src/main/java/org/dspace/content/dao/impl/MetadataFieldDAOImpl.java

```diff
@@ -71,6 +71,20 @@ public MetadataField findByElement(Context context, String metadataSchema, Strin
     }
 
     @Override
+    public List<MetadataField> findByUnqualifiedElement(Context context, String metadataSchema, String element) throws SQLException
+    {
+        Criteria criteria = createCriteria(context, MetadataField.class);
+        criteria.createAlias("metadataSchema", "s").add(
+                Restrictions.and(
+                        Restrictions.eq("s.name", metadataSchema),
+                        Restrictions.eq("element", element)
```

Comment:
We either need to change the DAO/service name here or the code itself. The name leads to believe that we want to retrieve all elements without a qualifier, but the query itself just queries the schema & element a "Restrictions.isNull("qualifier")" will be required to ensure a null qualifier.

If this method is meant to find all fields for a certain element then we should rename the method in the DAO & service. Having clear method names will prevent duplicate methods & makes it easier to figure out what the expected outcome should be.